### PR TITLE
Update clef version and amend configuration

### PR DESCRIPTION
**Pull request by significance**

### CODE REVIEW

File: docs/installation/bee-clef.md [hunk 1]
```diff
@@ -98,9 +98,11 @@ brew services start swarm-clef
 
 
 ### Configuring Bee Clef
-Configuration files are stored in `/etc/bee-clef/`
+Configuration files are stored in `/etc/default/bee-clef/` on Linux and `/usr/local/etc/swarm-clef/default` on MacOS.
 
-For a normal installation using a package manger, there should not be any configuration changes necessary to start using Bee Clef.
+To install clef for Swarm mainnet, change `BEE_CLEF_CHAIN_ID` to be `100` in order to interact with the XDAI network. For testnet, use chain id `5`.
+
+For a normal installation using a package manger, this should be the only configuration changes necessary to start using Bee Clef.
```
Comment (ralph-pichler) [inline docs/installation/bee-clef.md:105]: manger

File: docs/installation/docker.md [hunk 2]
```diff
@@ -118,12 +118,14 @@ BEE_SWAP_ENDPOINT=https://stake.getblock.io/mainnet/?api_key=your-api-key
 BEE_PASSWORD=my-password
 ```
 
-To enable Clef support, we must also change the following params: 
+To enable Clef support on mainnet, we must also change the following params: 
 
 ```
-CLEF_CHAINID=5
+BEE_CLEF_CHAIN_ID=100
```
Comment (ralph-pichler) [inline docs/installation/docker.md:124]: i think it is just `CLEF_CHAINID` for docker. please verify.

File: docs/installation/bee-clef.md [hunk 3]
```diff
@@ -68,14 +68,14 @@ sudo rpm -i bee-clef_0.4.13_amd64.rpm
 ##### ARMv7
 
 ```bash
-wget https://github.com/ethersphere/bee-clef/releases/download/v0.4.13/bee-clef_0.4.13_armv7.rpm
+wget https://github.com/ethersphere/bee-clef/releases/download/v0.5.0/bee-clef_0.4.13_armv7.rpm
```
Comment (ralph-pichler) [inline docs/installation/bee-clef.md:71]: there is still 0.4.13 in all the urls

File: docs/installation/rasp-bee-ry-pi.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ Raspberry Pi [https://docs.ethswarm.org](https://docs.ethswarm.org).
 Click `Install > Bee Clef` and scroll down to find the installation commands for Bee Clef **ARM (Raspberry Pi) ARMv7**. These should look something like: 
 
 ```sh
-wget https://github.com/ethersphere/bee-clef/releases/download/v0.4.13/bee-clef_0.4.13_armv7.deb
+wget https://github.com/ethersphere/bee-clef/releases/download/v0.5.0/bee-clef_0.4.13_armv7.deb
```
Comment (ralph-pichler) [inline docs/installation/rasp-bee-ry-pi.md:67]: this will probably also need the same chain id comments.

File: docs/installation/docker.md (hunk 2 above)
Comment (significance) [reply docs/installation/docker.md:124]: verified!

**Review by ralph-pichler — APPROVED** [review]

**Pull request merged**